co/connectionSet.cpp

```diff
@@ -608,29 +608,26 @@ bool ConnectionSet::_setupFDSet()
 
     // add regular connections
     _impl->lock.set();
-    for( ConnectionsCIter i = _impl->allConnections.begin();
-         i != _impl->allConnections.end(); ++i )
+    for( ConnectionPtr connection : _impl->allConnections )
     {
-        ConnectionPtr connectionPtr = *i;
-        const Connection& connection = *connectionPtr.get();
-        fd.fd = connection.getNotifier();
+        fd.fd = connection->getNotifier();
 
         if( fd.fd <= 0 )
         {
-            LBINFO << "Cannot select connection " << connectionPtr
-                   << ", connection " << typeid( connection ).name()
+            LBINFO << "Cannot select connection " << connection
+                   << ", connection " << lunchbox::className( connection.get( ))
```

Comment:
This may not print what you expect, before it was a reference and now it's a pointer.